zipkin-lens/src/components/TracePage/TraceSummary.jsx

```diff
@@ -94,15 +102,28 @@ const TraceSummary = React.memo(({ traceSummary }) => {
     return rerootedTree.filter(span => !allHiddenSpanIds[span.spanId]);
   }, [rerootedTree, childrenHiddenSpanIds]);
 
-  // Find the minumum and maximum timestamps in the shown spans.
-  const startTs = useMemo(() => minBy(rerootedTree, 'timestamp').timestamp, [rerootedTree]);
-  const endTs = useMemo(() => rerootedTree.map((span) => {
```

Comment:
If there are some spans which does not have timestamps, `endTs` will be `NaN`.
This is the cause of #2926 error.